sass/layout/_post.scss

```diff
@@ -1890,25 +1889,21 @@
     padding: 1px 4px;
 }
 
-.bot-indicator {
-    margin: 2px 10px 0 -4px;
-}
-
-.bot-indicator__autocomplete {
-    margin: 0px 4px;
-    padding: 0px 5px;
-    display: inline;
-}
-
-.bot-indicator__popoverlist {
-    margin: 0px 4px;
-    display: inline;
-}
-
-.bot-indicator__admin {
-    margin: 0px 4px;
-    background: rgba(61, 60, 64, 0.15);
-    display: inline;
+.Badge {
```

Comment:
This is outside the component because is the external part of the component, how it must be displayed because where I'm putting it. It allows to have a more agnostic component.